docs/configuration/pragmas.md

```diff
@@ -301,7 +301,7 @@ PRAGMA enable_profile;
 
 ##### Profiling Format
 
-The format of the resulting profiling information can be specified as either `json`, `query_tree`, or `query_tree_optimizer`. The default format is `query_tree`, which prints the physical operator tree together with the timings and cardinalities of each operator in the tree to the screen.
+The format of the resulting profiling information can be specified as either `json`, `query_tree`, or `query_tree_optimizer`. The default format is `query_tree`, which prints the logical query plan together with the timings and cardinalities of each operator in the tree to the screen.
```

Comment:
`physical query plan`, as we print the specific operators instead of the logical operations.